lib/matplotlib/tests/test_table.py

```diff
@@ -270,3 +270,24 @@ def test_table_dataframe(pd):
     for r, (index, row) in enumerate(df.iterrows()):
         for c, col in enumerate(df.columns if r == 0 else row.values):
             assert table[r if r == 0 else r+1, c].get_text().get_text() == str(col)
+
+
+# Test function for fontsize in table
+def test_table_fontsize():
+    # Data for plotting
+    tableData = [['a', 1], ['b', 1]]
+    # Create the figure and axis objects
+    fig, ax = plt.subplots()
+    # Plot the data (although we are focusing on the table)
+    ax.plot(np.linspace(0, 10, 100), np.linspace(0, 10, 100) + 1)
```

Comment:
This appears to be plotting a straight line that is unrelated to the table.  Can you explain why it is here?  If not, take it out.